README.md

```diff
@@ -61,26 +61,35 @@ cd ui && yarn start
 ## Using the Dapp
 1. `agoric open` will have opened your wallet at http://127.0.0.1:8000/
 
-1. `yarn start` will open a the Dapp at http://127.0.0.1:3000.
+2. `yarn start` will open a the Dapp at http://127.0.0.1:3000.
 
-1. In the wallet, `Accept` the `Dapp Connection` between cardStore and the wallet.
+3. Open a new tab from your and go to https://wallet.agoric.app/locator/ and enter http://127.0.0.1:8000/ into the 
+dialog as shown below.
 
-   ![Accept Dapp Connection](./readme-assets/accept-dapp-connection.png)
+    <br/><img width="50%" src="./readme-assets/locator.png">
 
-1. In the dapp, you should be able to click on a baseball card to `BID` on it in an action. Enter `Bid ammount` to submit an offer to buy the card.
+4. Now go to http://127.0.0.1:3000 and make sure you've enabled cookies for https://wallet.agoric.app/ as shown below.
```

Comment:
I feel enabling cookies is a bit of an advanced user flow. By default most users will have cookies enabled & those who don't know how to handle it.

If you would remove this bullet & image & renumber the bullets based on the removal I can approve.

Reply:
Can you check now?

Reply:
Looks great!